### DISCUSSION TITLE
Onion Query Language Null

### DISCUSSION THREAD

**Discussion by Ryan8bit** [Security-Onion-Solutions/securityonion#9496] [Unsupported Versions]
How do I specify in Onion Query Language (for the Hunt Interface queries) that the event.module is null?

!(event.module: ) | groupby event.module
does not work

**Comment by dougburks** [comment]
Is this what you're looking for?
```
-_exists_:event.module
```

**Reply by Ryan8bit** [reply]
If the event.module field is blank, is -_exists_:event.module true or false?

**Reply by dougburks** [reply]
If you search SOC for `-_exists_:event.module`, it should show logs that do NOT have the `event.module` field.

If you remove the hypen and search for `_exists_:event.module`, it should show logs that DO have the `event.module` field.

**Reply by Ryan8bit** [reply]
How do I remove items with a blank event.module field?

**Reply by dougburks** [reply]
Depending on your data, one option might be to `OR` the desired modules like this:
```
_exists_:event.module AND (event.module:module1 OR event.module:module2 OR event.module:module3)
```

**Reply by Ryan8bit** [reply]
This is useful. That option is a useful workaround. Thanks!
